libs/StatusGoQt/src/StatusGo/Messages/MessageDto.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+namespace Status::StatusGo::Messages
+{
+
+enum class ContentType
```

Comment:
It will be great to mention here the status-go equivalent for now. In the future, the plan is that this kind of common definition to be represented by protobuf and common to clients and status-go library. See [WalletApi.h](https://github.com/status-im/status-desktop/blob/26fcb8b99f5e761d2b9734bb5d2a7a01e9b78c40/libs/StatusGoQt/src/StatusGo/Wallet/WalletApi.h#L73) for an example